### PR TITLE
Potential fix for non-ascii-remote-errors

### PR DESCRIPTION
**Pull request by timbu**

### CODE REVIEW

File: test/test_exceptions.py [hunk 1]
```diff
@@ -70,6 +70,19 @@ def test_deserialize_to_remote_error():
...
+def test_deserialize_to_remote_error_with_non_ascii():
+
+    exc = CustomError(u'something w\xe9nt wrong')
+    data = serialize(exc)
+
+    deserialized = deserialize(data)
+    assert type(deserialized) == RemoteError
+    assert deserialized.exc_type == "CustomError"
+    assert deserialized.value == u"something w\xe9nt wrong"
+    assert repr(deserialized) == (
```
Comment (timbu) [inline test/test_exceptions.py]: `str` won't work on RemoteError here - does this matter?

**Comment by davidszotten** [comment]
is this not a duplicate of #262 ?

**Comment by timbu** [comment]
Indeed it is. Thought I'd checked the latest release, but obviously not... :)

**Comment by davidszotten** [comment]
not released yet. there's an rc branch

**Pull request closed without merging**